src/Umbraco.Web.Common/Security/MemberClaimsPrincipalFactory.cs

```diff
@@ -26,22 +26,32 @@ public class MemberClaimsPrincipalFactory : UserClaimsPrincipalFactory<MemberIde
     protected virtual string AuthenticationType => IdentityConstants.ApplicationScheme;
 
     /// <inheritdoc />
-    protected override async Task<ClaimsIdentity> GenerateClaimsAsync(MemberIdentityUser user)
+    protected override async Task<ClaimsIdentity> GenerateClaimsAsync(MemberIdentityUser member)
     {
         // Get the base
-        ClaimsIdentity baseIdentity = await base.GenerateClaimsAsync(user);
+        ClaimsIdentity baseIdentity = await base.GenerateClaimsAsync(member);
 
         // now create a new one with the correct authentication type
+
+        // NOTE: NameClaim is not Options.ClaimsIdentity.UserNameClaimType
+        // As we override the default from MS Identity to not be ClaimType.Name
+        // and be 'Umbraco.MemberUserName' to store the login/username warren@umbraco.com
+        // Then we can use the name claim type to store actual friendly member name
         var memberIdentity = new ClaimsIdentity(
             AuthenticationType,
-            Options.ClaimsIdentity.UserNameClaimType,
+            ClaimTypes.Name,
             Options.ClaimsIdentity.RoleClaimType);
 
+        // Explicitly set the Name claim to be the actual friendly member name
+        // and not the login/email of the member
+        // This means the identity above we are creating will assign User.Identity.Name correctly
+        memberIdentity.AddOrUpdateClaim(new Claim(ClaimTypes.Name, member.Name ?? "Unknown"));
+
```

Comment:
Still dont understand why this is not handled in the mapper from IMember to MemberIdentity

Reply:
Because no claims are there in the mapping when I was debugging/stepping through and it’s not a common extension point for Microsoft Identity and to me feels strange to put into some mapping.

Reply:
@bergmania for context there are 0 claims here, so for me this is still not the right place

![image](https://user-images.githubusercontent.com/1389894/194522926-fdef15c1-66bd-42cb-abe3-dad77287e196.png)


Also the branch has been updated to target `v11/dev`